### PR TITLE
Note about common update errors

### PR DESCRIPTION
**Pull request by kmadejski**
One of our customers experienced a problem with composer during the update process. This note might be helpful in the future.

### CODE REVIEW

File: docs/releases/updating_ez_platform.md [hunk 1]
```diff
@@ -120,6 +120,10 @@ On PHP conflict | 16.02 and later requires PHP 5.5 or higher
 
 Because from release 16.02 onwards eZ Platform is compatible only with PHP 5.5 and higher, the update command above will fail if you use an older PHP version. Please update PHP to proceed.
 
+!!! Common errors
```
Comment (DominikaK) [inline docs/releases/updating_ez_platform.md]: For the note box to generate correctly, you need to add a note type (note/tip/caution) and put the title in quotation marks:
`!!! note "Common errors"`

File: docs/releases/updating_ez_platform.md [hunk 2]
```diff
@@ -120,6 +120,10 @@ On PHP conflict | 16.02 and later requires PHP 5.5 or higher
...
+
+If you experienced issues during the update, please check [Common errors](https://doc.ezplatform.com/en/master/getting_started/about_composer/#cloning-failed-using-an-ssh-key) section.
```
Comment (DominikaK) [inline docs/releases/updating_ez_platform.md]: This sort of note box requires the box body to be indented by 4 spaces.
Also, you can use a relative link which will work for all versions: `(../getting_started/about_composer/#cloning-failed-using-an-ssh-key)`

**Comment by kmadejski** [comment]
@DominikaK fixed, review again please :)

File: docs/releases/updating_ez_platform.md [hunk 3]
```diff
@@ -120,6 +120,10 @@ On PHP conflict | 16.02 and later requires PHP 5.5 or higher
 
 Because from release 16.02 onwards eZ Platform is compatible only with PHP 5.5 and higher, the update command above will fail if you use an older PHP version. Please update PHP to proceed.
 
+!!! caution "Common errors"
+
+    If you experienced issues during the update, please check [Common errors](../getting_started/about_composer/#cloning-failed-using-an-ssh-key) section.
```
Comment (andrerom) [inline docs/releases/updating_ez_platform.md]: `section ` => `section on the about composer page.`

**Comment by kmadejski** [comment]
@andrerom fixed, thank you for the review.

**Pull request merged**